### DISCUSSION TITLE
DXCC incorrect in ClubLog

### DISCUSSION THREAD

**Discussion by eric-stammers-db** [magicbug/Cloudlog#1006] [General]
HI when my log is transferred to ClubLog the 'DXCC field is more than often recorded as 'SATELLITE, INTERNET OR REPEATER'. The last 4 entries all have the correct country. So what am i doing wrong?

**Comment by magicbug** [comment]
Bit confused about how DXCC Field can be Satellite? that is prop mode, certainly nothing wrong with Clublog code as I can see mine going in ok, require some more explanation on this issue to help

**Reply by 2e0wkw** [reply]
Here is a snip from my ClubLog all these entries are from CloudLog
![snip](https://user-images.githubusercontent.com/82234607/116780866-10865000-aa77-11eb-82fe-1db793773fed.jpg)

**Reply by 2e0wkw** [reply]
Hi, I contacted Michael at ClubLog who spotted a corner case in the ADIF upload and fixed it but this left my first 7 QSOs in this state. After looking at the details i realised i had not set the sat mode to S/X for QO-100. Once i had corrected that and imported it was all ok.

Thanks all for help

**Reply by magicbug** [reply]
Ah ! pleased its resolved.
